components/browser/menu/src/main/java/mozilla/components/browser/menu/BrowserMenu.kt

```diff
@@ -135,7 +135,6 @@ class BrowserMenu internal constructor(
     }
```

Comment:
(unrelated to this LOC)

I've been playing around with this API and I'm really loving it.  This solves just about every problem I can remember running into with the toolbar, and builds a strong framework for adding new functionality we might need in the future.